server/server/middlewares/mock.middleware.ts

```diff
@@ -31,7 +31,7 @@ export default class MockMiddleware {
         sniffer,
       );
 
-      if (mock != null && mock.isActive === true) {
+      if (mock != null && mock.isActive === true && sniffer.isMockingEnabled) {
```

Comment:
Please move this check to the previoes if statement.
If the mocking is disabled the code should not query for the mock and save some latency :)

Reply:
@sharkiodev: Nice catch, addressed this.

Reply:
Re-requested for review